docs/checkers/index.md

```diff
@@ -9,6 +9,29 @@ email to \[david at cs dot toronto dot edu\].
 These errors generally indicate a misuse of variables, control flow, or other Python features in our
 code.
 
+(W0134)=
+
+### Return in finally
+
+This error occurs when a `return` statement is used inside a `finally` block.
```

Comment:
It would be good to go into more detail about _why_ this is a problem (remember that the intended audience is students learning Python). You gave a great explanation in yesterday's meeting, and could write something similar here.

It's useful to use concrete examples to help with this. So, I would recommend that you refer to the example you created, e.g. "In the example below, ..."